src/main/java/appeng/block/crafting/BlockCraftingMonitor.java

```diff
@@ -1,6 +1,6 @@
 /*
  * This file is part of Applied Energistics 2.
- * Copyright (c) 2013 - 2014, AlgorithmX2, All rights reserved.
+ * Copyright (c) 2013 - 2015, AlgorithmX2, All rights reserved.
```

Comment:
Maybe just replace it once in every file and then again on every New Year's Day?